server/src/main/java/org/elasticsearch/monitor/jvm/HotThreads.java

```diff
@@ -117,6 +118,12 @@ public HotThreads type(ReportType type) {
         return this;
     }
 
+    // Used for testing
+    public HotThreads sunThreadInfo(SunThreadInfo sunThreadInfo) {
```

Comment:
Please remove this method as it is public and will be confusing to users of the HotThreads class inside Elasticsearch.